lib/db/schema/patch-059-060.sql

```diff
@@ -0,0 +1,232 @@
+CREATE PROCEDURE `createSessionToken_6` (
```

Comment:
These appear to be the same procedure as before the uaFormFactor. Could we call it directly inside of this, ex. `CALL createSessionToken_4(...)`? Not sure if it makes sense or is the right thing to do, but it could simplify this migration.